src/styles/variables.scss

```diff
@@ -27,10 +27,10 @@ $white: white;
 $green: #008000;
 $jungle-green: #29ab87;
 $active-green: #87d870;
-$yellow: #ecef08;
+$yellow: #eab308;
```

Comment:
Doesn't this apply to all the places not just task card ? Is that intentional?

Reply:
only one use. 
<img width="268" alt="image" src="https://github.com/Real-Dev-Squad/website-status/assets/61227144/99541d63-ec75-4e79-be9c-fe721418ca99">